src/glview/GLView.h

```diff
@@ -69,6 +71,10 @@ class GLView
   bool showedges;
   bool showcrosshairs;
   bool showscale;
+  GLdouble modelview[16];
+  GLdouble projection[16];
```

Comment:
This causes the build fail for WASM / HEADLESS. The simple, slightly ugly way for solving the issue is probably another entry in the NULLGL section of system-gl.h.